client/state/data-layer/wpcom/sites/rewind/api-transformer.js

```diff
@@ -36,7 +52,8 @@ const transformRewind = data =>
 			status: data.status,
 		},
 		data.progress && { progress: data.progress },
-		data.reason && { reason: data.reason }
+		data.reason && { reason: data.reason },
+		data.links && data.links.dismiss && { dismiss: makeRewindDismisser( data.links.dismiss ) }
```

Comment:
@samouri @jsnajdr @coderkevin thoughts on this?

We have the ability to provide a link in the API response which means Calypso doesn't need the logic and code to do it. This link turns into an action on the state in Redux (not persisting) and in this case sadly is a function that produces a Redux action. I'm still trying to figure out how best to remove this and I may end up doing it by passing the site id back in the API response.

Here above you can see how I'm using the `http()` action to send the request and then triggering a related update in response. Challenging in `http()` is that I can't transform the output of the API call to make the action. I can only statically provide the action (this of course was my plan in putting in the constraint so I'm eating my words though I think it may be a good thing still).